### PR TITLE
Update docker-library images

### PR DESCRIPTION
**Pull request by tianon**
- `docker`: 18.06.1-ce-rc2, `alpine:3.8` (docker-library/docker#124)
- `mariadb`: 10.3.9
- `mongo`: 4.1.2
- ~~`php`: 7.3-rc (docker-library/php#677), 7.2/alpine3.8 (docker-library/php#688)~~

### CODE REVIEW

File: library/php [hunk 1]
```diff
@@ -149,17 +199,17 @@ Architectures: amd64, arm32v5, arm32v7, i386
...
 Directory: 7.0/jessie/zts
 
-Tags: 7.0.31-cli-alpine3.7, 7.0-cli-alpine3.7, 7.0.31-alpine3.7, 7.0-alpine3.7, 7.0.31-cli-alpine, 7.0-cli-alpine, 7.0.31-alpine, 7.0-alpine
+Tags: 7.0.31-cli-alpine3.7, 7.0-cli-alpine3.7, 7.0.31-alpine3.7, 7.0-alpine3.7
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
 GitCommit: 2d413e2dff930b1ac261a587cb8911df2c6bd355
 Directory: 7.0/alpine3.7/cli
 
-Tags: 7.0.31-fpm-alpine3.7, 7.0-fpm-alpine3.7, 7.0.31-fpm-alpine, 7.0-fpm-alpine
+Tags: 7.0.31-fpm-alpine3.7, 7.0-fpm-alpine3.7
```
Comment (TimWolla) [inline library/php]: Is it intentional that the `7.0-fpm-alpine` tags have been removed (because there is no Alpine 3.8 for 7.0).

**Comment by TimWolla** [comment]
Check out whether you are interested in #4734 before running your local tests on this PR. :smile:

**Comment by tianon** [comment]
Removed `php` for now -- gotta resolve https://github.com/docker-library/php/pull/702 first (and probably should do #4734 too). :+1:

<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 68fea6d..c111be8 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -13,9 +13,9 @@ docker:18.06.0-ce-dind
 docker:18.06.0-ce-git
 docker:18.06.0-dind
 docker:18.06.0-git
-docker:18.06.1-ce-rc1
-docker:18.06.1-ce-rc1-dind
-docker:18.06.1-ce-rc1-git
+docker:18.06.1-ce-rc2
+docker:18.06.1-ce-rc2-dind
+docker:18.06.1-ce-rc2-git
 docker:dind
 docker:edge
 docker:edge-dind
@@ -49,12 +49,12 @@ mariadb:10.1.35
 mariadb:10.1.35-bionic
 mariadb:10.2
 mariadb:10.2-bionic
-mariadb:10.2.16
-mariadb:10.2.16-bionic
+mariadb:10.2.17
+mariadb:10.2.17-bionic
 mariadb:10.3
 mariadb:10.3-bionic
-mariadb:10.3.8
-mariadb:10.3.8-bionic
+mariadb:10.3.9
+mariadb:10.3.9-bionic
 mariadb:bionic
 mariadb:latest
 mongo:3
@@ -116,12 +116,12 @@ mongo:4.1-windowsservercore-1709
 mongo:4.1-windowsservercore-1803
 mongo:4.1-windowsservercore-ltsc2016
 mongo:4.1-xenial
-mongo:4.1.1
-mongo:4.1.1-windowsservercore
-mongo:4.1.1-windowsservercore-1709
-mongo:4.1.1-windowsservercore-1803
-mongo:4.1.1-windowsservercore-ltsc2016
-mongo:4.1.1-xenial
+mongo:4.1.2
+mongo:4.1.2-windowsservercore
+mongo:4.1.2-windowsservercore-1709
+mongo:4.1.2-windowsservercore-1803
+mongo:4.1.2-windowsservercore-ltsc2016
+mongo:4.1.2-xenial
 mongo:latest
 mongo:unstable
 mongo:unstable-windowsservercore
diff --git a/_bashbrew-shared-tags b/_bashbrew-shared-tags
index e4635e7..cd54e77 100644
--- a/_bashbrew-shared-tags
+++ b/_bashbrew-shared-tags
@@ -20,8 +20,8 @@ mongo:4.0.1 -- mongo:xenial, mongo:windowsservercore-ltsc2016, mongo:windowsserv
 mongo:4.0.1-windowsservercore -- mongo:windowsservercore-ltsc2016, mongo:windowsservercore-1709, mongo:windowsservercore-1803
 mongo:4.1 -- mongo:unstable-xenial, mongo:unstable-windowsservercore-ltsc2016, mongo:unstable-windowsservercore-1709, mongo:unstable-windowsservercore-1803
 mongo:4.1-windowsservercore -- mongo:unstable-windowsservercore-ltsc2016, mongo:unstable-windowsservercore-1709, mongo:unstable-windowsservercore-1803
-mongo:4.1.1 -- mongo:unstable-xenial, mongo:unstable-windowsservercore-ltsc2016, mongo:unstable-windowsservercore-1709, mongo:unstable-windowsservercore-1803
-mongo:4.1.1-windowsservercore -- mongo:unstable-windowsservercore-ltsc2016, mongo:unstable-windowsservercore-1709, mongo:unstable-windowsservercore-1803
+mongo:4.1.2 -- mongo:unstable-xenial, mongo:unstable-windowsservercore-ltsc2016, mongo:unstable-windowsservercore-1709, mongo:unstable-windowsservercore-1803
+mongo:4.1.2-windowsservercore -- mongo:unstable-windowsservercore-ltsc2016, mongo:unstable-windowsservercore-1709, mongo:unstable-windowsservercore-1803
 mongo:latest -- mongo:xenial, mongo:windowsservercore-ltsc2016, mongo:windowsservercore-1709, mongo:windowsservercore-1803
 mongo:unstable -- mongo:unstable-xenial, mongo:unstable-windowsservercore-ltsc2016, mongo:unstable-windowsservercore-1709, mongo:unstable-windowsservercore-1803
 mongo:unstable-windowsservercore -- mongo:unstable-windowsservercore-ltsc2016, mongo:unstable-windowsservercore-1709, mongo:unstable-windowsservercore-1803
diff --git a/docker_latest/Dockerfile b/docker_latest/Dockerfile
index 5cf8621..d7ba95a 100644
--- a/docker_latest/Dockerfile
+++ b/docker_latest/Dockerfile
@@ -1,4 +1,4 @@
-FROM alpine:3.7
+FROM alpine:3.8
 
 RUN apk add --no-cache \
 		ca-certificates
diff --git a/docker_test/Dockerfile b/docker_test/Dockerfile
index a83084c..b71d3f2 100644
--- a/docker_test/Dockerfile
+++ b/docker_test/Dockerfile
@@ -1,4 +1,4 @@
-FROM alpine:3.7
+FROM alpine:3.8
 
 RUN apk add --no-cache \
 		ca-certificates
@@ -10,7 +10,7 @@ RUN apk add --no-cache \
 RUN [ ! -e /etc/nsswitch.conf ] && echo 'hosts: files dns' > /etc/nsswitch.conf
 
 ENV DOCKER_CHANNEL test
-ENV DOCKER_VERSION 18.06.1-ce-rc1
+ENV DOCKER_VERSION 18.06.1-ce-rc2
 # TODO ENV DOCKER_SHA256
 # https://github.com/docker/docker-ce/blob/5b073ee2cf564edee5adca05eee574142f7627bb/components/packaging/static/hash_files !!
 # (no SHA file artifacts on download.docker.com yet as of 2017-06-07 though)
diff --git a/mariadb_10.2/Dockerfile b/mariadb_10.2/Dockerfile
index d065ee2..1cdf922 100644
--- a/mariadb_10.2/Dockerfile
+++ b/mariadb_10.2/Dockerfile
@@ -90,7 +90,7 @@ RUN set -ex; \
 
 # bashbrew-architectures: amd64 arm64v8 ppc64le
 ENV MARIADB_MAJOR 10.2
-ENV MARIADB_VERSION 1:10.2.16+maria~bionic
+ENV MARIADB_VERSION 1:10.2.17+maria~bionic
 
 RUN set -e;\
 	echo "deb http://ftp.osuosl.org/pub/mariadb/repo/$MARIADB_MAJOR/ubuntu bionic main" > /etc/apt/sources.list.d/mariadb.list; \
diff --git a/mariadb_latest/Dockerfile b/mariadb_latest/Dockerfile
index 0567f77..2cd0a8a 100644
--- a/mariadb_latest/Dockerfile
+++ b/mariadb_latest/Dockerfile
@@ -90,7 +90,7 @@ RUN set -ex; \
 
 # bashbrew-architectures: amd64 arm64v8 ppc64le
 ENV MARIADB_MAJOR 10.3
-ENV MARIADB_VERSION 1:10.3.8+maria~bionic
+ENV MARIADB_VERSION 1:10.3.9+maria~bionic
 
 RUN set -e;\
 	echo "deb http://ftp.osuosl.org/pub/mariadb/repo/$MARIADB_MAJOR/ubuntu bionic main" > /etc/apt/sources.list.d/mariadb.list; \
diff --git a/mongo_unstable-windowsservercore-1709/Dockerfile b/mongo_unstable-windowsservercore-1709/Dockerfile
index 79cf3f9..0d7d7d2 100644
--- a/mongo_unstable-windowsservercore-1709/Dockerfile
+++ b/mongo_unstable-windowsservercore-1709/Dockerfile
@@ -2,9 +2,9 @@ FROM microsoft/windowsservercore:1709
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
 
-ENV MONGO_VERSION 4.1.1
-ENV MONGO_DOWNLOAD_URL https://downloads.mongodb.org/win32/mongodb-win32-x86_64-2012plus-4.1.1-signed.msi
-ENV MONGO_DOWNLOAD_SHA256 ff14a5e670543a966500d562195dcdd9738b3dd76cb0122b4b3648215e6cf47d
+ENV MONGO_VERSION 4.1.2
+ENV MONGO_DOWNLOAD_URL https://downloads.mongodb.org/win32/mongodb-win32-x86_64-2012plus-4.1.2-signed.msi
+ENV MONGO_DOWNLOAD_SHA256 4e327fa11d290fc755b92b9c8d16102f0f3984c6bee3da5630839c2a6448dd6e
 
 RUN Write-Host ('Downloading {0} ...' -f $env:MONGO_DOWNLOAD_URL); \
 	(New-Object System.Net.WebClient).DownloadFile($env:MONGO_DOWNLOAD_URL, 'mongo.msi'); \
diff --git a/mongo_unstable-windowsservercore-1803/Dockerfile b/mongo_unstable-windowsservercore-1803/Dockerfile
index 3b8f5c4..54f3229 100644
--- a/mongo_unstable-windowsservercore-1803/Dockerfile
+++ b/mongo_unstable-windowsservercore-1803/Dockerfile
@@ -2,9 +2,9 @@ FROM microsoft/windowsservercore:1803
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
 
-ENV MONGO_VERSION 4.1.1
-ENV MONGO_DOWNLOAD_URL https://downloads.mongodb.org/win32/mongodb-win32-x86_64-2012plus-4.1.1-signed.msi
-ENV MONGO_DOWNLOAD_SHA256 ff14a5e670543a966500d562195dcdd9738b3dd76cb0122b4b3648215e6cf47d
+ENV MONGO_VERSION 4.1.2
+ENV MONGO_DOWNLOAD_URL https://downloads.mongodb.org/win32/mongodb-win32-x86_64-2012plus-4.1.2-signed.msi
+ENV MONGO_DOWNLOAD_SHA256 4e327fa11d290fc755b92b9c8d16102f0f3984c6bee3da5630839c2a6448dd6e
 
 RUN Write-Host ('Downloading {0} ...' -f $env:MONGO_DOWNLOAD_URL); \
 	(New-Object System.Net.WebClient).DownloadFile($env:MONGO_DOWNLOAD_URL, 'mongo.msi'); \
diff --git a/mongo_unstable-windowsservercore-ltsc2016/Dockerfile b/mongo_unstable-windowsservercore-ltsc2016/Dockerfile
index 86160f7..f5ef9da 100644
--- a/mongo_unstable-windowsservercore-ltsc2016/Dockerfile
+++ b/mongo_unstable-windowsservercore-ltsc2016/Dockerfile
@@ -2,9 +2,9 @@ FROM microsoft/windowsservercore:ltsc2016
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
 
-ENV MONGO_VERSION 4.1.1
-ENV MONGO_DOWNLOAD_URL https://downloads.mongodb.org/win32/mongodb-win32-x86_64-2012plus-4.1.1-signed.msi
-ENV MONGO_DOWNLOAD_SHA256 ff14a5e670543a966500d562195dcdd9738b3dd76cb0122b4b3648215e6cf47d
+ENV MONGO_VERSION 4.1.2
+ENV MONGO_DOWNLOAD_URL https://downloads.mongodb.org/win32/mongodb-win32-x86_64-2012plus-4.1.2-signed.msi
+ENV MONGO_DOWNLOAD_SHA256 4e327fa11d290fc755b92b9c8d16102f0f3984c6bee3da5630839c2a6448dd6e
 
 RUN Write-Host ('Downloading {0} ...' -f $env:MONGO_DOWNLOAD_URL); \
 	(New-Object System.Net.WebClient).DownloadFile($env:MONGO_DOWNLOAD_URL, 'mongo.msi'); \
diff --git a/mongo_unstable-xenial/Dockerfile b/mongo_unstable-xenial/Dockerfile
index 5f873cb..8112099 100644
--- a/mongo_unstable-xenial/Dockerfile
+++ b/mongo_unstable-xenial/Dockerfile
@@ -66,7 +66,7 @@ ARG MONGO_REPO=repo.mongodb.org
 ENV MONGO_PACKAGE=${MONGO_PACKAGE} MONGO_REPO=${MONGO_REPO}
 
 ENV MONGO_MAJOR 4.1
-ENV MONGO_VERSION 4.1.1
+ENV MONGO_VERSION 4.1.2
 
 RUN echo "deb http://$MONGO_REPO/apt/ubuntu xenial/${MONGO_PACKAGE%-unstable}/$MONGO_MAJOR multiverse" | tee "/etc/apt/sources.list.d/${MONGO_PACKAGE%-unstable}.list"
 
````

</details>

**Comment by tianon** [comment]
Build test of #4733; 3276ee69; `amd64` (`docker`, `mariadb`, `mongo`):

```console

$ bashbrew build docker:18.06.1-ce-rc2
Using bashbrew/cache:8768806f5bf77c7977ce150926fc4d1211f9fb71d9f9b60407912b904b15042e (docker:18.06.1-ce-rc2)
Tagging docker:18.06.1-ce-rc2
Tagging docker:18.06-rc
Tagging docker:rc
Tagging docker:test

$ test/run.sh docker:18.06.1-ce-rc2
testing docker:18.06.1-ce-rc2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build docker:18.06.1-ce-rc2-dind
Using bashbrew/cache:94dcbeeb388a58bc72f9ac0517c536ded32069096308d7da817377b33991d149 (docker:18.06.1-ce-rc2-dind)
Tagging docker:18.06.1-ce-rc2-dind
Tagging docker:18.06-rc-dind
Tagging docker:rc-dind
Tagging docker:test-dind

$ test/run.sh docker:18.06.1-ce-rc2-dind
testing docker:18.06.1-ce-rc2-dind
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'docker-dind' [5/6]...passed
	'docker-registry-push-pull' [6/6]...passed


$ bashbrew build docker:18.06.1-ce-rc2-git
Using bashbrew/cache:b965752a36b429330007e9569c2d519d4b04f8fa6b22f33dedc2fcf9aabed178 (docker:18.06.1-ce-rc2-git)
Tagging docker:18.06.1-ce-rc2-git
Tagging docker:18.06-rc-git
Tagging docker:rc-git
Tagging docker:test-git

$ test/run.sh docker:18.06.1-ce-rc2-git
testing docker:18.06.1-ce-rc2-git
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build docker:18.06.0-ce
Using bashbrew/cache:fc9cee27e119ed553c48da9c14b6fc25419e15642b91d4e8936e0b8ef34812a4 (docker:18.06.0-ce)
Tagging docker:18.06.0-ce
Tagging docker:18.06.0
Tagging docker:18.06
Tagging docker:18
Tagging docker:edge
Tagging docker:stable
Tagging docker:latest

$ test/run.sh docker:18.06.0-ce
testing docker:18.06.0-ce
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build docker:18.06.0-ce-dind
Using bashbrew/cache:b100cd669754ae6c08dab277de852f613cc09abafbad7860fe74174d436f6fd8 (docker:18.06.0-ce-dind)
Tagging docker:18.06.0-ce-dind
Tagging docker:18.06.0-dind
Tagging docker:18.06-dind
Tagging docker:18-dind
Tagging docker:edge-dind
Tagging docker:stable-dind
Tagging docker:dind

$ test/run.sh docker:18.06.0-ce-dind
testing docker:18.06.0-ce-dind
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'docker-dind' [5/6]...passed
	'docker-registry-push-pull' [6/6]...passed


$ bashbrew build docker:18.06.0-ce-git
Using bashbrew/cache:8cb024a08a7c1a2e678b79cb08d63d8007ae8b95df93c27a790b8ca37a1eca4f (docker:18.06.0-ce-git)
Tagging docker:18.06.0-ce-git
Tagging docker:18.06.0-git
Tagging docker:18.06-git
Tagging docker:18-git
Tagging docker:edge-git
Tagging docker:stable-git
Tagging docker:git

$ test/run.sh docker:18.06.0-ce-git
testing docker:18.06.0-ce-git
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

```console

$ bashbrew build mariadb:10.3.9-bionic
Using bashbrew/cache:27e36155aacac3d2b4e418cc59f507791773521c32711140cf120cd4c24e23a7 (mariadb:10.3.9-bionic)
Tagging mariadb:10.3.9-bionic
Tagging mariadb:10.3-bionic
Tagging mariadb:10-bionic
Tagging mariadb:bionic
Tagging mariadb:10.3.9
Tagging mariadb:10.3
Tagging mariadb:10
Tagging mariadb:latest

$ test/run.sh mariadb:10.3.9-bionic
testing mariadb:10.3.9-bionic
	'utc' [1/7]...passed
	'cve-2014--shellshock' [2/7]...passed
	'no-hard-coded-passwords' [3/7]...passed
	'override-cmd' [4/7]...passed
	'mysql-basics' [5/7]......passed
	'mysql-initdb' [6/7]......passed
	'mysql-log-bin' [7/7]......passed


$ bashbrew build mariadb:10.2.17-bionic
Using bashbrew/cache:a592b03eca4b52cf4da725985d54a11d8a83e8bd750f828294613b593f27de6e (mariadb:10.2.17-bionic)
Tagging mariadb:10.2.17-bionic
Tagging mariadb:10.2-bionic
Tagging mariadb:10.2.17
Tagging mariadb:10.2

$ test/run.sh mariadb:10.2.17-bionic
testing mariadb:10.2.17-bionic
	'utc' [1/7]...passed
	'cve-2014--shellshock' [2/7]...passed
	'no-hard-coded-passwords' [3/7]...passed
	'override-cmd' [4/7]...passed
	'mysql-basics' [5/7]......passed
	'mysql-initdb' [6/7]......passed
	'mysql-log-bin' [7/7]......passed


$ bashbrew build mariadb:10.1.35-bionic
Using bashbrew/cache:dab512ac94bac78c9f09cd0d7ce48d465e82d32e4ab73050b26ca51855462e40 (mariadb:10.1.35-bionic)
Tagging mariadb:10.1.35-bionic
Tagging mariadb:10.1-bionic
Tagging mariadb:10.1.35
Tagging mariadb:10.1

$ test/run.sh mariadb:10.1.35-bionic
testing mariadb:10.1.35-bionic
	'utc' [1/7]...passed
	'cve-2014--shellshock' [2/7]...passed
	'no-hard-coded-passwords' [3/7]...passed
	'override-cmd' [4/7]...passed
	'mysql-basics' [5/7].........passed
	'mysql-initdb' [6/7].........passed
	'mysql-log-bin' [7/7].........passed


$ bashbrew build mariadb:10.0.36-xenial
Using bashbrew/cache:1ef9c1c2ed43427f5a8ba6c3942a3b02282a873713c1a28d31c18dd9f910ac6d (mariadb:10.0.36-xenial)
Tagging mariadb:10.0.36-xenial
Tagging mariadb:10.0-xenial
Tagging mariadb:10.0.36
Tagging mariadb:10.0

$ test/run.sh mariadb:10.0.36-xenial
testing mariadb:10.0.36-xenial
	'utc' [1/7]...passed
	'cve-2014--shellshock' [2/7]...passed
	'no-hard-coded-passwords' [3/7]...passed
	'override-cmd' [4/7]...passed
	'mysql-basics' [5/7].......passed
	'mysql-initdb' [6/7].......passed
	'mysql-log-bin' [7/7].......passed


$ bashbrew build mariadb:5.5.61-trusty
Using bashbrew/cache:36606d40d402fb0c55db3593bcf465e1de255cce5545374bc8eadb3acfdeec74 (mariadb:5.5.61-trusty)
Tagging mariadb:5.5.61-trusty
Tagging mariadb:5.5-trusty
Tagging mariadb:5-trusty
Tagging mariadb:5.5.61
Tagging mariadb:5.5
Tagging mariadb:5

$ test/run.sh mariadb:5.5.61-trusty
testing mariadb:5.5.61-trusty
	'utc' [1/7]...passed
	'cve-2014--shellshock' [2/7]...passed
	'no-hard-coded-passwords' [3/7]...passed
	'override-cmd' [4/7]...passed
	'mysql-basics' [5/7]......passed
	'mysql-initdb' [6/7]......passed
	'mysql-log-bin' [7/7]......passed

```

```console

$ bashbrew build mongo:3.2.20-jessie
Using bashbrew/cache:c460e90e95bb48f88892936066957b6deb65d7ac3f4c872d0580e244dd26993f (mongo:3.2.20-jessie)
Tagging mongo:3.2.20-jessie
Tagging mongo:3.2-jessie
Tagging mongo:3.2.20
Tagging mongo:3.2

$ test/run.sh mongo:3.2.20-jessie
testing mongo:3.2.20-jessie
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'mongo-basics' [5/6]...passed
	'mongo-auth-basics' [6/6]....passed


$ bashbrew build mongo:3.2.20-windowsservercore-ltsc2016
skipping "mongo:3.2.20-windowsservercore-ltsc2016" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:3.2.20-windowsservercore-ltsc2016
testing mongo:3.2.20-windowsservercore-ltsc2016
	image does not exist!


$ bashbrew build mongo:3.2.20-windowsservercore-1709
skipping "mongo:3.2.20-windowsservercore-1709" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:3.2.20-windowsservercore-1709
testing mongo:3.2.20-windowsservercore-1709
	image does not exist!


$ bashbrew build mongo:3.4.16-jessie
Using bashbrew/cache:cf848e953863dc8325c98ddb45523e985d956398ac8961c8a22db06d31ff2264 (mongo:3.4.16-jessie)
Tagging mongo:3.4.16-jessie
Tagging mongo:3.4-jessie
Tagging mongo:3.4.16
Tagging mongo:3.4

$ test/run.sh mongo:3.4.16-jessie
testing mongo:3.4.16-jessie
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'mongo-basics' [5/6]...passed
	'mongo-auth-basics' [6/6]....passed


$ bashbrew build mongo:3.4.16-windowsservercore-ltsc2016
skipping "mongo:3.4.16-windowsservercore-ltsc2016" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:3.4.16-windowsservercore-ltsc2016
testing mongo:3.4.16-windowsservercore-ltsc2016
	image does not exist!


$ bashbrew build mongo:3.4.16-windowsservercore-1709
skipping "mongo:3.4.16-windowsservercore-1709" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:3.4.16-windowsservercore-1709
testing mongo:3.4.16-windowsservercore-1709
	image does not exist!


$ bashbrew build mongo:3.6.6-jessie
Using bashbrew/cache:0859b6a1c70f4b0cf299fedf8c489faeda6b94df4dd2c90343c6351f092f790d (mongo:3.6.6-jessie)
Tagging mongo:3.6.6-jessie
Tagging mongo:3.6-jessie
Tagging mongo:3-jessie
Tagging mongo:3.6.6
Tagging mongo:3.6
Tagging mongo:3

$ test/run.sh mongo:3.6.6-jessie
testing mongo:3.6.6-jessie
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'mongo-basics' [5/6]....passed
	'mongo-auth-basics' [6/6]....passed


$ bashbrew build mongo:3.6.6-windowsservercore-ltsc2016
skipping "mongo:3.6.6-windowsservercore-ltsc2016" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:3.6.6-windowsservercore-ltsc2016
testing mongo:3.6.6-windowsservercore-ltsc2016
	image does not exist!


$ bashbrew build mongo:3.6.6-windowsservercore-1709
skipping "mongo:3.6.6-windowsservercore-1709" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:3.6.6-windowsservercore-1709
testing mongo:3.6.6-windowsservercore-1709
	image does not exist!


$ bashbrew build mongo:4.0.1-xenial
Using bashbrew/cache:60a46716eba624bab8d0763eea95a38923fcf58f69f0215ae74531a18d5b0546 (mongo:4.0.1-xenial)
Tagging mongo:4.0.1-xenial
Tagging mongo:4.0-xenial
Tagging mongo:4-xenial
Tagging mongo:xenial
Tagging mongo:4.0.1
Tagging mongo:4.0
Tagging mongo:4
Tagging mongo:latest

$ test/run.sh mongo:4.0.1-xenial
testing mongo:4.0.1-xenial
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'mongo-basics' [5/6]....passed
	'mongo-auth-basics' [6/6]....passed


$ bashbrew build mongo:4.0.1-windowsservercore-ltsc2016
skipping "mongo:4.0.1-windowsservercore-ltsc2016" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:4.0.1-windowsservercore-ltsc2016
testing mongo:4.0.1-windowsservercore-ltsc2016
	image does not exist!


$ bashbrew build mongo:4.0.1-windowsservercore-1709
skipping "mongo:4.0.1-windowsservercore-1709" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:4.0.1-windowsservercore-1709
testing mongo:4.0.1-windowsservercore-1709
	image does not exist!


$ bashbrew build mongo:4.0.1-windowsservercore-1803
skipping "mongo:4.0.1-windowsservercore-1803" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:4.0.1-windowsservercore-1803
testing mongo:4.0.1-windowsservercore-1803
	image does not exist!


$ bashbrew build mongo:4.1.2-xenial
Using bashbrew/cache:ecc86492049146a34debf3a4998cc9b36aa682602495e8a8996162b2e37b9cf2 (mongo:4.1.2-xenial)
Tagging mongo:4.1.2-xenial
Tagging mongo:4.1-xenial
Tagging mongo:unstable-xenial
Tagging mongo:4.1.2
Tagging mongo:4.1
Tagging mongo:unstable

$ test/run.sh mongo:4.1.2-xenial
testing mongo:4.1.2-xenial
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'mongo-basics' [5/6]....passed
	'mongo-auth-basics' [6/6]....passed


$ bashbrew build mongo:4.1.2-windowsservercore-ltsc2016
skipping "mongo:4.1.2-windowsservercore-ltsc2016" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:4.1.2-windowsservercore-ltsc2016
testing mongo:4.1.2-windowsservercore-ltsc2016
	image does not exist!


$ bashbrew build mongo:4.1.2-windowsservercore-1709
skipping "mongo:4.1.2-windowsservercore-1709" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:4.1.2-windowsservercore-1709
testing mongo:4.1.2-windowsservercore-1709
	image does not exist!


$ bashbrew build mongo:4.1.2-windowsservercore-1803
skipping "mongo:4.1.2-windowsservercore-1803" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh mongo:4.1.2-windowsservercore-1803
testing mongo:4.1.2-windowsservercore-1803
	image does not exist!

```


The following images failed at least one test:

- `mongo`:
  - `mongo:3.2.20-windowsservercore-ltsc2016`
  - `mongo:3.2.20-windowsservercore-1709`
  - `mongo:3.4.16-windowsservercore-ltsc2016`
  - `mongo:3.4.16-windowsservercore-1709`
  - `mongo:3.6.6-windowsservercore-ltsc2016`
  - `mongo:3.6.6-windowsservercore-1709`
  - `mongo:4.0.1-windowsservercore-ltsc2016`
  - `mongo:4.0.1-windowsservercore-1709`
  - `mongo:4.0.1-windowsservercore-1803`
  - `mongo:4.1.2-windowsservercore-ltsc2016`
  - `mongo:4.1.2-windowsservercore-1709`
  - `mongo:4.1.2-windowsservercore-1803`

**Pull request merged**